requirements.txt

```diff
@@ -0,0 +1,118 @@
+anyio==4.0.0
```

Comment:
Did you generate this file with a `pip freeze`?

Reply:
yes, do you mean for requirement.txt.